Source/Core/DolphinQt/Debugger/WatchWidget.cpp

```diff
@@ -158,34 +164,30 @@ void WatchWidget::Update()
                                               QLatin1Char('0')));
         decimal->setText(QString::number(PowerPC::HostRead_U32(entry.address)));
         string->setText(QString::fromStdString(PowerPC::HostGetString(entry.address, 32)));
+        floatValue->setText(QString::number(PowerPC::HostRead_F32(entry.address)));
       }
     }
 
     address->setForeground(brush);
+    string->setFlags(Qt::ItemIsEnabled);
 
     int column = 0;
 
-    for (auto* item : {label, address, hex, decimal, string})
+    for (auto* item : items)
```

Comment:
This could be `for (int column = 0; column < NUM_COLUMNS; column++)` then `m_table->setItem(i, column, items[column];`.

Reply:
Done.